buildSrc/src/main/groovy/com/uber/okbuck/core/model/base/Scope.groovy

```diff
@@ -54,17 +56,27 @@ class Scope {
         jvmArgs = jvmArguments
         this.depCache = depCache
 
-        extractConfigurations(configurations)
+        extractConfiguration(configuration)
```

Comment:
lets guard against null here since we already have the deps sets initialized to be empty

Reply:
done